hello-core-logic/createListMimicTrello.js

```diff
@@ -0,0 +1,41 @@
+//  Create List
+// Write a function named createList that takes two arguments, boardName and listName. The function should create a list with name
+// listName within board boardName with no cards to start. If the boardName doesn't exist, it should return Board doesn't exist
+// and, if a list with the same name exists already, it should return List name already exists.
+
+const hello = {
+    'Tester Board': {
+      'To Do': ['Laundry', 'Buy Apples', 'Pay Phone Bill'],
+      'Doing': ['Laundry', 'Studying Javascript', 'Studying HTML', 'Studying Ruby'],
+      'Done': ['Laundry']
+    },
+    'Dreams': {
+      'Wish List': ['Conquer the Seven Kingdoms', 'Get my baby back', 'My hand needs to chill'],
+    },
+    'Cuisine': {
+      'Region':['African', 'European', 'Brazilian', 'Asian'] }
+};
+
+board = "Cuisine";
```

Comment:
make sure that when you define ANY and ALL your variables that you include one of the javascript keywords: `let` or `const`